YAFC/Workspace/ProductionTable/ProductionTableFlatHierarchy.cs

```diff
@@ -4,10 +4,15 @@
 using YAFC.UI;
 
 namespace YAFC {
+    /// <summary>
```

Comment:
Thank you for documenting the code! We need more of that. It reduces the amount of effort the next reader has to do.